rhp/v4/rpc_test.go

```diff
@@ -266,10 +267,44 @@ func TestFormContractBasis(t *testing.T) {
 	n, genesis := testutil.V2Network()
 	hostKey, renterKey := types.GeneratePrivateKey(), types.GeneratePrivateKey()
 
-	cm, s, w := startTestNode(t, n, genesis)
+	cm, s, _ := startTestNode(t, n, genesis)
+
+	// create a separate wallet that doesn't automatically sync with the test node
+	wCS, wState, err := chain.NewDBStore(chain.NewMemDB(), n, genesis)
+	if err != nil {
+		t.Fatal(err)
+	}
+	wCM := chain.NewManager(wCS, wState)
+	wStore := testutil.NewEphemeralWalletStore()
+	w, err := wallet.NewSingleAddressWallet(renterKey, wCM, wStore)
```

Comment:
This whole bit can probably still use `startTestNode`, just don't connect the syncs